spec/retrier_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Error < StandardError; end
```

Comment:
Defining the class in the global namespace will leave it defined for other tests as well which might lead to confusion